clicache/src/Region.hpp

```diff
@@ -26,7 +26,7 @@
 
 #include "IRegion.hpp"
 #include "ISubscriptionService.hpp"
-#include "native_shared_ptr.hpp"
+#include "native_conditional_shared_ptr.hpp"
```

Comment:
I wonder if we really have anywhere left that the CLI needs to have a `shared_ptr` to the `Region`. There should be no place where the `Region` would outlive the `Cache` so I think that any public API that gets the `Region` shouldn't have any ownership responsibilities.